alws/utils/modularity.py

```diff
@@ -131,6 +131,7 @@ def generate_new_context(self) -> str:
         hashes = '{0}:{1}'.format(build_context, runtime_context)
         return hashlib.sha1(hashes.encode('utf-8')).hexdigest()[:8]
 
+    # TODO: find out why this function is needed
```

Comment:
This is needed to override dependencies in the template itself

Reply:
But we don't override anything, just because we have this condition:
https://github.com/AlmaLinux/albs-web-server/blob/4fee3769e80e574866f9c9f411ebba68e1c187b8/alws/utils/modularity.py#L158
we can add only stream to old dependencies if it doesn't exist